engine/src/main/java/io/camunda/zeebe/engine/processing/common/ElementActivationBehavior.java

```diff
@@ -280,4 +296,25 @@ private void createEventSubscriptions(
       }
     }
   }
+
+  public static class ActivatedElementKeys {
```

Comment:
🔧 Class can be private

Reply:
Lies! We can't make this private as it's being used in the `ProcessInstanceModificationProcessor` 😉

Reply:
Haha, woops 😆